dd-sdk-android-internal/src/main/java/com/datadog/android/internal/profiler/BenchmarkCounter.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.internal.profiler
+
+import com.datadog.tools.annotation.NoOpImplementation
+
+/**
+ * Interface of benchmark counter to be implemented to perform count metrics.
+ * This should only used by internal benchmarking.
+ */
+@NoOpImplementation
+interface BenchmarkCounter
```

Comment:
why do we have an interface without any methods? Is it useful to have such at the current point?

Reply:
It's reserved for the future development according to the RFC, but you are right we can remove it since we don't have idea what are the methods inside.